### PR TITLE
Live stream controls

### LINKED ISSUE #135: Add Live Stream Controls component
[Design specs](https://www.figma.com/file/ikwcMcE6HEb653b0oVCgy6/%E2%9A%99%EF%B8%8FKajabi-Live-SAGE?node-id=2%3A0)

Screen Shot 2020-04-02 at 2 35 20 PM

### PR DESCRIPTION
**Pull request by philschanely**
Closes #135 

Screen Shot 2020-04-02 at 2 36 34 PM

See Objects > Live Stream Controls

### CODE REVIEW

**Review by pixelflips — APPROVED** [review]
👍 +1 - Nice work!

File: app/assets/stylesheets/sage/system/vendor/_reboot.scss [hunk 1]
```diff
@@ -55,7 +56,7 @@ h6 {
   margin-bottom: 0;
 }
 
-p {
+p, li {
   margin-top: 0;
   margin-bottom: 0;
 }
```
Comment (AndrwM) [inline app/assets/stylesheets/sage/system/vendor/_reboot.scss:62]: Thanks for adding this!

File: app/views/sage/examples/objects/live_stream_control/_preview.html.erb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        <i class="sage-live-stream-control__icon sage-icon-stop"></i>
+        <span class="visually-hidden">Stop</span>
+    </button>
+
+    <!-- Pill variation -->
+    <button class="sage-live-stream-control sage-live-stream-control--pill">
+        <i class="sage-live-stream-control__icon sage-icon-users"></i>
+        <span class="sage-live-stream-control__label">324</span>
+    </button>
+</div>
```
Comment (AndrwM) [inline app/views/sage/examples/objects/live_stream_control/_preview.html.erb:34]: 4 space indentions in this file

**Review by AndrwM — APPROVED** [review]
Sweeet, looking forward to seeing this in action. +1

**Pull request merged**